src/index.ts

```diff
@@ -66,12 +65,12 @@ export const dotAbcDotBonkOrDotPoorToWallet = async (
 // Docs for this suck, so check out
 // https://github.com/onsol-labs/tld-parser/blob/main/tests/tld-parser.spec.ts#L97
 // getMainDomain() is what we want
-export const walletToDotAbcDotBonkOrDotPoor = async (
+export const walletToDotAnything = async (
   connection: Connection,
   wallet: PublicKey
 ): Promise<WalletNameAndProfilePicture> => {
   const parser = new TldParser(connection);
-  let mainDomain: MainDomain;
+  let mainDomain: any;
```

Comment:
We generally try and avoid `any`. Should this be `string | null`?

Reply:
we don't export the type and for the moment it could remain `any`

Reply:
I'm going to update the https://www.npmjs.com/package/@onsol/tldparser lib today so we can go back to using MainDomain

Reply:
updated the tldParser lib it now exports the MainDomain type so this is fixed now